libs/design/src/core/openable/openable.directive.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Directive,
+  EventEmitter,
+  HostBinding,
+  Input,
+  Output,
+} from '@angular/core';
+
+import { DaffOpenable } from './openable';
+
+@Directive({
+  selector: '[daffOpenable]',
+  standalone: true,
+})
+
+export class DaffOpenableDirective implements DaffOpenable {
+  @Input() @HostBinding('class.daff-open') open = false;
+
+  @Output() toggled: EventEmitter<boolean> = new EventEmitter<boolean>();
```

Comment:
What are your thoughts on `opened`? I know we originally said no to this, but I really dislike `toggled`.

Reply:
I think we said no because this output handles opened or closed, so calling it opened is incorrect

Reply:
`toggled` does seem more accurate than `opened`. The only other idea I have is `openedUpdate` or `openedChange`

Reply:
decided to stick with `toggled`